README.md

```diff
@@ -257,16 +257,29 @@ bemLinter({
 ### Defining a component
 
 The plugin will only run if it finds special comments that
-define a named component or a group of utilities.
+define a named component or a group of utilities,
+or the implicit config option is set.
```

Comment:
There needs to be more documentation describing what an `implicit` definition means .